src/main/java/ml/shifu/shifu/core/dtrain/wdl/WDLParallelGradient.java

```diff
@@ -143,6 +165,17 @@ public WDLParams doCompute() {
         return params;
     }
 
+    public WDLParams doCompute(int iteration, int trainBatchSize, int validateBatchSize) {
+        if(validateBatchSize < 10) {
+            LOG.info("Origin validationBatchSize is " + validateBatchSize + " which is so less and adjust to a large value");
```

Comment:
Could it be warning or add verification in ModelProbe...?

Reply:
Add ModelProbe for WDL